ui/app/send.js

```diff
@@ -351,6 +382,15 @@ SendTransactionScreen.prototype.setActiveCurrency = function (newCurrency) {
   this.setState({ activeCurrency: newCurrency })
 }
 
+SendTransactionScreen.prototype.estimateGasAndPrice = function () {
```

Comment:
After refactoring the action to return promise and use local state, can you also add this in send token as well?